parquet-column/src/test/java/org/apache/parquet/column/values/bloomfilter/TestBlockSplitBloomFilter.java

```diff
@@ -181,6 +182,83 @@ public void testBloomFilterNDVs(){
     assertTrue(bytes < 5 * 1024 * 1024);
   }
 
+  @Test
+  public void testMergeEmptyBloomFilter() throws IOException {
```

Comment:
Probably we need a test to expect throwing exception when two BFs are not compatible.

Reply:
I added a test for two BFs are not compatible.